cli/compose/types/types.go

```diff
@@ -125,6 +126,18 @@ type ServiceConfig struct {
 	WorkingDir      string `mapstructure:"working_dir"`
 }
 
+// BuildConfig is a type for build
+// using the same format at libcompose: https://github.com/docker/libcompose/blob/master/yaml/build.go#L12
+type BuildConfig struct {
+	Context    string
+	Dockerfile string
+	Args       MappingWithEquals
+	Labels     Labels
+	CacheFrom  StringList
```

Comment:
This one will need a `mapstructure:` annotation because it defaults to using a dash instead of an underscore to separate the words.